nagios.go

```diff
@@ -26,3 +49,22 @@ func GetNagiosHistoricalData(r Runner, project, pod string) Task {
 		return r.Run(cmd, path)
 	}
 }
+
+// getResourceNamesBySubstr returns a list of names for the provided resource type that contain
+// the provided string, in the provided project.
+func getResourceNamesBySubstr(project, resource, substr string) ([]string, error) {
```

Comment:
maybe we need a new file for shared functions and that? Like `shared.go` or something?